src/desktop/analytics/inquiry_questionnaire.js

```diff
@@ -1,23 +1,23 @@
 import { data as sd } from "sharify"
 import {
-  createdAccount,
-  successfullyLoggedIn,
   ContextModule,
   Intent,
+  createdAccount,
+  successfullyLoggedIn,
 } from "@artsy/cohesion"
 import $ from "jquery"
 import { omit } from "lodash"
 const analyticsHooks = require("desktop/lib/analytics_hooks.coffee")
 const analytics = window.analytics
-;(function() {
+;(function () {
   "use strict"
 
-  var namespace, track, trackWithoutNamespace, bind, bindOnce
+  let namespace, track, trackWithoutNamespace, bind, bindOnce
 
   function getTrackingOptions() {
-    var trackingOptions = {}
+    let trackingOptions = {}
 
-    const referrer = window.analytics.__artsyReferrer
+    const referrer = window.analytics.__artsyClientSideRoutingReferrer
```

Comment:
Only change in this file (prettier / eslint churn) -- updated this variable name to be more descriptive